icu4j/main/classes/core/src/com/ibm/icu/impl/number/LongNameHandler.java

```diff
@@ -303,4 +311,26 @@ public Modifier getModifier(Signum signum, StandardPlural plural) {
         // Signum ignored
         return modifiers.get(plural);
     }
+
+    private static final class PluralTableSink extends UResource.Sink {
```

Comment:
Optional: I prefer if you wouldn't move code around for no reason.

Reply:
In this case not optional, in my opinion: moving code such that documentation is no longer accurate will make things harder for future readers of the code. I'm moving everything that wasn't modified, back to where it was.

Reply:
With commit 08744e0867 I have un-done the moves, whitespace changes and indent changes I found that seemed to serve no purpose.

I've also undone the import changes, even if the import order in these files aren't following desired style. I'd prefer a style fix PR to have only style fixes, no logic changes - much easier to review.